packages/webapp/webapp_server.js

```diff
@@ -307,34 +305,11 @@ function getBoilerplateAsync(request, arch) {
     });
   });
 
-  return promise.then(() => {
-    const useMemoized = ! (
-      data.dynamicHead ||
-      data.dynamicBody ||
-      madeChanges
```

Comment:
The reason I left the memoization in place when I originally implemented the `server-render` package was because it could be better for performance when no server-side rendering is happening.

That said, I think it's important to make sure the no-cache path is fast enough, since we want people to do SSR, and we want SSR to be fast.

Assuming we want to remove the memoization completely, I think there are a few more opportunities for simplification. In particular, there's no point in worrying about the `madeChanges` variable, and we can stop pre-generating boilerplate elsewhere in this module, e.g. here:
```js
WebAppInternals.setBundledJsCssUrlRewriteHook = function (hookFn) {
  bundledJsCssUrlRewriteHook = hookFn;
  WebAppInternals.generateBoilerplate();
};
```

Reply:
I see now why it's hard to memoize a stream, so I've come around to these changes. 👌